Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -74,7 +84,7 @@ public Vector3 ManipulationHandPosition
         {
             get
             {
-                Vector3 handPosition = Vector3.zero;
```

Comment:
The method is called TryGetPosition so without checking its implementation it looks like it's possible to not get a "correct" position, is that correct ? I guess that's why the code was originally setting it to zero.
I guess it would be safer to check for the returned Boolean and return zero if TryGetPosition returned false.